--- vmmaker.cmake	2025-03-11 18:47:33.252201418 +0100
+++ vmmaker.cmake.new	2025-03-11 18:48:17.831398862 +0100
@@ -120,7 +120,7 @@
         ExternalProject_Add(
             vmmaker
 
-            URL https://files.pharo.org/image/120/Pharo12.0-SNAPSHOT.build.1519.sha.aa50f9c.arch.64bit.zip
+            URL /build/repository/build/build/Pharo12.0-SNAPSHOT.build.1519.sha.aa50f9c.arch.64bit.zip
             URL_HASH SHA256=b12270631ffc0c6adcb0b6449565b9abfd8e88a863a894a7320f660c05a0af1e
             BUILD_COMMAND ${VMMAKER_VM} --headless ${VMMAKER_DIR}/image/Pharo12.0-SNAPSHOT-64bit-aa50f9c.image --no-default-preferences save VMMaker
 	    COMMAND ${VMMAKER_VM} --headless ${VMMAKER_IMAGE} --no-default-preferences --save --quit "${CMAKE_CURRENT_SOURCE_DIR_TO_OUT}/scripts/installVMMaker.st" "${CMAKE_CURRENT_SOURCE_DIR_TO_OUT}" "${ICEBERG_DEFAULT_REMOTE}"
